tests/integration/conftest.py

```diff
@@ -499,11 +534,26 @@ def _setup_ontology(project):
 
 
 @pytest.fixture
-def configured_project_with_complex_ontology(client, rand_gen, image_url):
+def configured_project_with_complex_ontology(client, initial_dataset, rand_gen,
+                                             image_url):
     project = client.create_project(name=rand_gen(str),
-                                    queue_mode=QueueMode.Dataset)
-    dataset = client.create_dataset(name=rand_gen(str), projects=project)
+                                    queue_mode=QueueMode.Batch,
+                                    media_type=MediaType.Image)
+    dataset = initial_dataset
     data_row = dataset.create_data_row(row_data=image_url)
+    data_row_ids = [data_row.uid]
+
+    project._wait_until_data_rows_are_processed(
+        data_row_ids=data_row_ids,
+        wait_processing_max_seconds=DATA_ROW_PROCESSING_WAIT_TIMEOUT_SECONDS,
+        sleep_interval=DATA_ROW_PROCESSING_WAIT_SLEEP_INTERNAL_SECONDS)
```

Comment:
the below `create_batch` function calls `_wait_until_data_rows_are_processed` internally, so no need for this here